### DISCUSSION TITLE
New IP redirecting to old IP

### DISCUSSION THREAD

**Discussion by yula-21** [Security-Onion-Solutions/securityonion#7598] [Unsupported Versions] [answered]
Hello,
I’ve changed my machine’s IP address, everything works fine in the network but when I use the web interface it sometimes redirects me to the old IP address.

At first it redirected me to the old IP address even when accessing the login page, that was fixed by changing the “url_base” field in the global.sls file.
But now I still get redirects when trying to access a pdf file I put in the motd.

Does anyone know why it happens? What other configuration files do I need to change?

**Comment by dougburks** [comment, marked as answer]
https://docs.securityonion.net/en/2.3/ip.html

**Reply by yula-21** [reply]
And if I have several interfaces? Will it ask me which interface to change?

I saw this command on your documentation when searching for an answer, I think it would help a lot if you add more information (like the answer to my question for example).
Especially since it says the command is experimental, its scary to use it when you have zero information about it.

**Reply by dougburks** [reply]
Yes, we intentionally placed a scary warning on that page as our official recommendation is to make sure that your IP address is correct during installation and then avoid changing it after installation:
https://docs.securityonion.net/en/2.3/best-practices.html#configuration

**Reply by yula-21** [reply]
Ok, but I have a testing environment for the development so I need to change the IP address when stepping out of the testing environment. The vm goes through a lot of configuration which makes it near impossible and irresponsible to install from scratch each and every time.

Im asking you how to use the command because it has no information in the documentation or explain in which files the ip is present so I can change it.
Directing me to the documentation again would do no help, I am asking here because I tried understanding and reading myself and couldn't find an answer online.

Thank you.

**Reply by dougburks** [reply]
Take a look at the contents of `/usr/sbin/so-ip-update`. It should be fairly self-explanatory.

**Reply by yula-21** [reply]
Thank you!
I checked and the script checks for all instances of the old ip address in /etc and /opt/so/saltstack using the following command:

`grep -rlI $OLD_IP /opt/so/saltstack /etc`

So you can use this command to see what you need to change if you don't trust the command or in my case if you have multiple network adapters and as a result of that multiple ip addresses.
